config/config.json.ctmpl

```diff
@@ -11,7 +11,8 @@
 	"musicbrainz": {
 		"clientID": "{{template "KEY" "musicbrainz/client_id"}}",
 		"clientSecret": "{{template "KEY" "musicbrainz/client_secret"}}",
-		"callbackURL": "{{template "KEY" "musicbrainz/callback_url"}}"
+		"callbackURL": "{{template "KEY" "musicbrainz/callback_url"}}",
+        "oauthTokenURL": "https://metabrainz.org/oauth2/token"
```

Comment:
I think this would be best added to a separate `oauth` object (with separate client id/secret) rather than adding it to `musicbrainz`, mainly because we are moving user authentication from MusicBrainz to MetaBrainz.
This copied what is being put in place for LB, for the record (in the private repo that feeds this template).

Let's also make sure we can change this value by using the same templating language as above.
So something like:

```suggestion
"oauth": {
		"clientID": "{{template "KEY" "oauth/client_id"}}",
		"clientSecret": "{{template "KEY" "oauth/client_secret"}}",
		"tokenURL": "{{template "KEY" "oauth/token_url"}}",
		"notificationURL": "{{template "KEY" "oauth/notification_url"}}"
	},
```